packages/functional-tests/tests/misc/keyStretchingV2.spec.ts

```diff
@@ -105,10 +106,15 @@ test.describe('key-stretching-v2', () => {
         await page.goto(
           `${target.contentServerUrl}/?showReactApp=true&forceExperiment=generalizedReactApp&forceExperimentGroup=react&${stretch}`
         );
-        await signupReact.fillOutEmailFirst(email);
+
+        await signupReact.fillOutEmailForm({ email: email, submit: true });
         await page.waitForSelector('#root');
```

Comment:
Not sure if this should use locator instead of selector, but noticed we had that `waitForRoot()` helper in the other file.
But also the code is unchanged, so non-blocking nit.

Reply:
It should be using the `waitForRoot()` you are right, I missed it in the cascade change.
This file is still left to be audited, would it be alright if I fix it in the next PR?

Reply:
Yeah, works for me. Possibly just file a new ticket and check these at the same time:
```sh
git grep -Pin "#root" | grep "waitForSelector"

packages/functional-tests/pages/resetPassword.ts:33:      const header = await this.page.waitForSelector('#root .card-header');
packages/functional-tests/tests/misc/keyStretchingV2.spec.ts:111:        await page.waitForSelector('#root');
packages/functional-tests/tests/misc/keyStretchingV2.spec.ts:206:        await page.waitForSelector('#root');
packages/functional-tests/tests/misc/keyStretchingV2.spec.ts:215:        await page.waitForSelector('#root');
packages/functional-tests/tests/react-conversion/cannotCreateAccount.spec.ts:17:    await page.waitForSelector('#root');
packages/functional-tests/tests/react-conversion/forceAuth.spec.ts:34:    await page.waitForSelector('#root');
packages/functional-tests/tests/react-conversion/forceAuth.spec.ts:61:    await page.waitForSelector('#root');
packages/functional-tests/tests/react-conversion/oauthResetPassword.spec.ts:358:    await page.waitForSelector('#root');
packages/functional-tests/tests/react-conversion/recoveryKey.spec.ts:80:      await page.waitForSelector('#root');
packages/functional-tests/tests/react-conversion/recoveryKey.spec.ts:96:      await page.waitForSelector('#root');
packages/functional-tests/tests/react-conversion/resetPassword.spec.ts:30:      await page.waitForSelector('#root');
packages/functional-tests/tests/react-conversion/resetPassword.spec.ts:53:      await page.waitForSelector('#root');
packages/functional-tests/tests/react-conversion/resetPassword.spec.ts:105:      await page.waitForSelector('#root');
packages/functional-tests/tests/react-conversion/resetPassword.spec.ts:129:      await page.waitForSelector('#root');
packages/functional-tests/tests/react-conversion/resetPassword.spec.ts:179:      // await page.waitForSelector('#root');
packages/functional-tests/tests/react-conversion/resetPassword.spec.ts:193:      await page.waitForSelector('#root');
packages/functional-tests/tests/react-conversion/resetPassword.spec.ts:217:      await page.waitForSelector('#root');
packages/functional-tests/tests/react-conversion/resetPassword.spec.ts:231:      await page.waitForSelector('#root');
packages/functional-tests/tests/react-conversion/resetPassword.spec.ts:240:      await page.waitForSelector('#root');
packages/functional-tests/tests/react-conversion/signinTotp.spec.ts:41:      await page.waitForSelector('#root');
packages/functional-tests/tests/react-conversion/syncV3ResetPassword.spec.ts:34:      await page.waitForSelector('#root');
```